compiler/main/version_num.h

```diff
@@ -23,7 +23,7 @@
 
 #define MAJOR_VERSION 1
 #define MINOR_VERSION 23
-#define UPDATE_VERSION 0
+#define UPDATE_VERSION 1
```

Comment:
Further down in this file, you _may_ want to change `officialRelease` to `false` as well, since this is not one.  Though that'll require other changes as well (because, e.g., the way versions are printed out will change).

Reply:
(Re-reading your OP, I see that this may have been intentional, which is fine with me).

Reply:
Yes, that's correct, and indeed there were some other changes I didn't pull in that were related to whether or not this was an "official release" branch.